### ISSUE TITLE
Z-index when dragging

### ISSUE THREAD

**Issue by gavinbelson** [rcdexta/react-trello#165]
When dragging a card from first lane to second lane. The title of the 2nd lane overlaps the card while dragging.

For example, when using the dataset in the examples as below. Drag Card2 into the Completed lane. While dragging it from lane to lane, drag it over the "Completed" text. The text will overlap the card.

```
      const data = {
        lanes: [
          {
            id: 'lane1',
            title: 'Planned Tasks',
            label: '2/2',
            cards: [
              {id: 'Card1', title: 'Write Blog', description: 'Can AI make memes', label: '30 mins'},
              {id: 'Card2', title: 'Pay Rent', description: 'Transfer via NEFT', label: '5 mins', metadata: {sha: 'be312a1'}}
            ]
          },
          {
            id: 'lane2',
            title: 'Completed',
            label: '0/0',
            cards: []
          }
        ]
      } 
```

**Comment by tatosjb** [comment]
As a work around, i have overited the this css rule and noted a post-it to make a pr for it

```css
  .board div span section header {
    z-index: auto;
  }
```

**Comment by gavinbelson** [comment]
Hmm.. I tried your suggestion and all kinds of variations with:

```
. div span section header {
    z-index: 9000 !important;
  }
.smooth-dnd-draggable-wrapper section header{
    z-index: 1 !important;
}
.react-trello-card {
    z-index: 9999 !important;
}

```
Still getting the lane header overlapping the card, when moving the card from one lane to another..

**Comment by rcdexta** [comment]
This is fixed in 2.0.10. Pls try again and let me know if problem persists for you...
